checkpoint_schedules/schedule.py

```diff
@@ -534,7 +480,7 @@ def is_running(self):
         -------
         bool
             Indicate whether the :class:`CheckpointSchedule` schedule has an 
-            iterator object. Do not finalise the checkpoint schedule if ``True``.
+            iterator object. Do not finalise the checkpoint schedule if `'True'`.
```

Comment:
Running schedules can be finalized. `is_running` returns whether at least one action has been yielded.